src/modules/rppt_tensor_audio_augmentations.cpp

```diff
@@ -186,3 +190,42 @@ RppStatus rppt_resample_host(RppPtr_t srcPtr,
         return RPP_ERROR_NOT_IMPLEMENTED;
     }
 }
+
+/********************************************************************************************************************/
+/*********************************************** RPP_GPU_SUPPORT = ON ***********************************************/
+/********************************************************************************************************************/
+
+#ifdef GPU_SUPPORT
+
+/******************** pre_emphasis_filter ********************/
+
+RppStatus rppt_pre_emphasis_filter_gpu(RppPtr_t srcPtr,
+                                       RpptDescPtr srcDescPtr,
+                                       RppPtr_t dstPtr,
+                                       RpptDescPtr dstDescPtr,
+                                       RpptImagePatchPtr srcDims,
```

Comment:
I think there is a problem with this PR
There is no argument of type RpptImagePatchPtr in declaration
https://github.com/sampath1117/rpp/blob/sr/pre_emphasis_filter_hip/include/rppt_tensor_audio_augmentations.h#L98

@HazarathKumarM i think this PR wont be able to build
Please confirm

Reply:
one of the commit didn't get push. I pushed all the commits now